cypress/support/commands.js

```diff
@@ -23,3 +23,14 @@
 //
 // -- This will overwrite an existing command --
 // Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
+
+Cypress.Commands.add('userInfo', () => {
```

Comment:
This command only returns a hardcoded user's information. Your data should be generated where it is possible